lib/compat/wordpress-6.5/fonts/fonts.php

```diff
@@ -157,7 +159,36 @@ function gutenberg_register_font_collections() {
 	if ( null !== WP_Font_Library::get_instance()->get_font_collection( 'google-fonts' ) ) {
 		return;
 	}
-	wp_register_font_collection( 'google-fonts', 'https://s.w.org/images/fonts/17.7/collections/google-fonts-with-preview.json' );
+	wp_register_font_collection(
+		'google-fonts',
+		array(
+			'name'          => _x( 'Google Fonts', 'font collection name' ),
```

Comment:
All of these strings need the 'gutenberg' text domain as well.

Reply:
added here: https://github.com/WordPress/gutenberg/pull/59256/commits/450f9f5302df7f25cfd4c88c3abc64fb9ce31180